2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_setup.py:_flush():71] setting env: {}
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220204_111403-1kellc3r/logs/debug.log
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220204_111403-1kellc3r/logs/debug-internal.log
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_init.py:init():404] calling init triggers
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-04 11:14:03,926 INFO    MainThread:17483 [wandb_init.py:init():460] starting backend
2022-02-04 11:14:03,926 INFO    MainThread:17483 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-04 11:14:03,944 INFO    MainThread:17483 [backend.py:ensure_launched():216] starting backend process...
2022-02-04 11:14:03,959 INFO    MainThread:17483 [backend.py:ensure_launched():222] started backend process with pid: 17520
2022-02-04 11:14:03,960 INFO    MainThread:17483 [wandb_init.py:init():469] backend started and connected
2022-02-04 11:14:03,963 INFO    MainThread:17483 [wandb_init.py:init():533] updated telemetry
2022-02-04 11:14:04,001 INFO    MainThread:17483 [wandb_init.py:init():563] communicating current version
2022-02-04 11:14:04,566 INFO    MainThread:17483 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-04 11:14:04,566 INFO    MainThread:17483 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-04 11:14:04,791 INFO    MainThread:17483 [wandb_init.py:init():606] starting run threads in backend
2022-02-04 11:14:06,268 INFO    MainThread:17483 [wandb_run.py:_console_start():1810] atexit reg
2022-02-04 11:14:06,268 INFO    MainThread:17483 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-04 11:14:06,268 INFO    MainThread:17483 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-04 11:14:06,270 INFO    MainThread:17483 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-04 11:14:06,270 INFO    MainThread:17483 [wandb_init.py:init():633] run started, returning control to user process
2022-02-04 11:14:08,785 INFO    MainThread:17483 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-04 11:14:08,786 INFO    MainThread:17483 [wandb_run.py:_restore():1752] restore
2022-02-04 11:14:11,243 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-02-04 11:14:11,519 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 790
}

2022-02-04 11:14:11,798 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 816
}

2022-02-04 11:14:11,900 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12227
}

2022-02-04 11:14:12,003 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12227
}

2022-02-04 11:14:12,106 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 790
  total_bytes: 12227
}

2022-02-04 11:14:12,208 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:12,311 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:12,413 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:12,515 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:12,617 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:12,719 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:13,207 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}

2022-02-04 11:14:13,718 INFO    MainThread:17483 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12227
  total_bytes: 12227
}
local_info {
}

2022-02-04 11:14:14,798 INFO    MainThread:17483 [wandb_run.py:_append_files():2180] logging synced files
